test/jasmine/tests/parcoords_test.js

```diff
@@ -1496,8 +1496,7 @@ describe('parcoords constraint interactions', function() {
         .then(delay(noSnapDelay))
         .then(function() {
             expect(getDashArray(1)).toBeCloseToArray(newDashArray);
-            // TODO: ideally this would get clipped to [0, 9]...
```

Comment:
@archmoj do you have any idea why someone wrote that it would have been "ideal" to have `[0, 9]` here?

Reply:
Since the range was `computed from pixel space` the bottom range was not exactly zero i.e. `-0.1020`.

Reply:
got it. Thanks for the info.